### DISCUSSION TITLE
go 依赖包升级方法

### DISCUSSION THREAD

**Discussion by giscus** [edony-ink/comments#2] [General]
# go 依赖包升级方法

在维护一个有一定体量的 go 项目的时候，难免会碰到需要依赖包管理的问题。关于 go 的依赖包管理有很多不错的文章介绍How to Use Go Modules，这篇文章总结的是项目于中 go mod 依赖包升级的问题。 背景go 项目中使用 go mod vendor 来离线管理所有的依赖包（主要是出于依赖版本锁定的目的），在项目日常管理中会涉及到几个跟依赖包升级的场景： 公共依赖包升级私有依赖包升级下面我们就分这两个场景来讨论。 公共依赖包升级公共开源的依赖包升级相对比较简单，在 go 项目下执行： go get -u github.com/xxx/xxx 更细致的还会有如下几种方式升级： 升级到指定版本: go get -u github.com/xxx/xxx@v1.6.2升级到指定 commit 的版本：go

https://www.edony.ink/go-mod-upgrade/

**Comment by edonyzpc** [comment]
excerpt 抓取失败？
